modules/ROOT/pages/troubleshooting.adoc

```diff
@@ -4,7 +4,137 @@ include::_attributes.adoc[]
+
+. Ensure the locale of the approver user configured in the `SelfRegister` component is set to English.
+
+Or:
+
+. Open Salesforce setup.
+. Open company settings.
+. Open language settings.
+. Enable the appropriate languages and locales by enabling end-user languages, enabling platform-only languages, or both.
+. Add the language matching the approver user profile to the list of displayed languages.
+
+== Not Authorized or Problem Logging In Error When Saving External Data Source Configuration
+
+If saving the external data source configuration shows one of the errors "Not authorized" or "Problem Logging In - No_Oauth_State: State was not sent back":
+
+. Ensure the connected app domain is the right domain of the Salesforce organization.
+. Open Salesforce setup.
+. Open the authentication providers configuration.
+. Click the name of the Anypoint authentication provider and ensure it has these settings from the correct connected app in Anypoint.
+  . Ensure the consumer key and consumer secret match the connected app in Anypoint.
+  . Ensure the authorize endpoint URL, token endpoint URL, and default scopes are configured as described in the section xref:configure-acm.adoc#create-your-authentication-provider[Create Your Authentication Provider] on the page Configure ACM.
+  . Ensure the test-only initialization URL, OAuth-only initialization URL, callback URL, and single logout URL match the correct domain of your Salesforce organization.
+  . Ensure your connected app in Anypoint has the correct domain of your Salesforce organization in the `redirect_uri` setting, client ID and client secret that match those in Salesforce, and correct scopes.
+
+To find details of the connected app, check the UI or make this API call:
+
+----
+curl -X GET 'https://anypoint.mulesoft.com/accounts/api/connectedApplications/' \
+   -H "Content-Type: application/json" \
+   -H "authorization: bearer XXXXXXXXX"
+----
+
+This call returns a response like this:
+
+---- 
+{
+"client_id": "5fafXXXXXXXXXXXXXXXXXXXXXXXX29c9",
+"client_secret": "9509XXXXXXXXXXXXXXXXXXXXXXXXC10E",
+"client_name": "ACM Integration Client",
+"redirect_uris": [
+  "https://YourOrgDomain.my.salesforce.com/services/authcallback/Anypoint"
+],
+"grant_types": [
+  "authorization_code",
+  "refresh_token"
+],
+"public_keys": [],
+"scopes": [
+  "offline_access",
+  "full"
+],
+"enabled": true,
+"owner_org_id": "f377XXXX-XXXX-XXXX-XXXX-XXXXXXXX9d08",
+"as_id": "anypoint"
+}
+----
+
+"YourOrgDomain" is replaced by your organization domain.
+
+== Missing Request Access Button
+
+If a published API version is not showing the *Request access* button:
+
+. Ensure the API version is managed in API Manager. If it is not, click *Import from Exchange*.
+. Ensure the API version has at least one endpoint defined.
+. Label the endpoint. This label displays in your community.
+. Ensure at least one of the API version's endpoints is configured as public in Exchange.
+
```

Comment:
add: 
"In the community builder, check the audience targeting settings of the API Request access component (LINK TO COMPONENT DOCS). If you want it to appear regardless of the user, make sure it is not assigned to any audience. If, alternatively, you want it to appear only to a specific audience, make sure that it is assigned to the correct audience and that you're verifying it with a user member of this audience."

Reply:
Added. Thank you!